### PR TITLE
Consensus dashboard validators card

### PR DESCRIPTION
**Pull request by buberdds**
Are we fine with merging all TODOs and sync data in the future when API is ready?

example with real request to Nexus, but with fake image and cumulative voting value

Screenshot from 2024-01-10 17-51-24

### CODE REVIEW

File: src/app/components/Validators/index.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        // TODO: Enable link when validator detail page is ready
+        content: (
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <>
+              {/* @ts-expect-error Property 'image' does not exist on type 'ValidatorMedia' */}
+              {validator.media?.image && (
+                <StyledImage
+                  alt={validator.media?.name || validator.entity_address}
+                  // @ts-expect-error Property 'image' does not exist on type 'ValidatorMedia'
+                  src={validator.media?.image}
```
Comment (buberdds) [inline src/app/components/Validators/index.tsx:77]: https://github.com/oasisprotocol/explorer/pull/1147#discussion_r1454326054

File: src/app/components/Validators/index.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    left: 0,
+    top: `-${theme.spacing(4)}`,
+  },
+}))
+
+export const Validators: FC<ValidatorsProps> = ({ isLoading, limit, pagination, validators }) => {
+  const { t } = useTranslation()
+
+  const tableColumns: TableColProps[] = [
+    { align: TableCellAlign.Center, key: 'rank', content: t('validator.rank') },
```
Comment (buberdds) [inline src/app/components/Validators/index.tsx:47]: common.rank already exists

File: src/app/components/Validators/index.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  marginRight: 15,
+})
+
+const StyledBox = styled(Box, {
+  shouldForwardProp: prop => prop !== 'value',
+})<{ value: number }>(({ value, theme }) => ({
+  position: 'relative',
+  textAlign: 'center',
+  '&::before': {
+    content: '""',
```
Comment (lukaw3d) [inline src/app/components/Validators/index.tsx:32]: suspicious

Comment (buberdds) [reply src/app/components/Validators/index.tsx:32]: content: '',
`Error: You seem to be using a value for 'content' without quotes, try replacing it with `content: '""`

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**